secure/storage/src/id.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::{Deserialize, Serialize};
+
+/// Id represents a Libra internal identifier for a given process. For example, safety_rules or
+/// key_manager. It is up to the Storage and its deployment to translate these identifiers into
+/// verifiable material. For example, the process running safety_rules may have a token that is
+/// intended for only safety_rules to own. The specifics are left to the implementation of the
+/// storage backend interface layer.
+#[derive(Debug, Deserialize, Serialize)]
+pub enum Id {
+    Anyone,
```

Comment:
`Anyone` and `NoOne` seems a `Permission` variant instead of a `Id` variant to me, curious what's your thought.

Reply:
Good call. Will move to another Enum called Permission.